internal/engine/buildcontrol/build_control.go

```diff
@@ -21,15 +21,15 @@ func NextTargetToBuild(state store.EngineState) *store.ManifestTarget {
 
 	// Local targets aren't parallelizable with any other kind of target.
 	// So if we're already building a local target, bail immediately.
-	if IsBuildingLocalTarget(state) {
+	if IsBuildingUnparallelizableLocalTarget(state) {
 		return nil
 	}
 
 	targets := state.Targets()
 	if IsBuildingAnything(state) {
-		// Local targets aren't parallelizable with any other kind of target.
+		// Some local targets aren't parallelizable with any other kind of target.
 		// So if we're already building any targets, remove all the local ones.
```

Comment:
```suggestion
		// Some local targets aren't parallelizable with any other kind of target.
```
just since "all the local ones" is no longer accurate, a better wording doesn't come immediately to mind, and the function name does a pretty good job of communicating this anyway, might as well delete this second line

Reply:
reworded a bit